cmd/skopeo/proxy.go

```diff
@@ -736,6 +788,8 @@ func (h *proxyHandler) processRequest(readBytes []byte) (rb replyBuf, terminate
 		rb, err = h.GetFullConfig(req.Args)
 	case "GetBlob":
 		rb, err = h.GetBlob(req.Args)
+	case "LayerInfoJSON":
```

Comment:
A naming nit here...I think the fact that we transport via JSON is an implementation detail.  How about just "GetLayerInfo"?

Reply:
Good idea.